neqo-transport/src/connection.rs

```diff
@@ -1641,6 +1694,9 @@ impl Connection {
             }
         }
         self.handle_lost_packets(&lost_packets);
+        if epoch == 3 {
+            self.maybe_discard_handshake_keys(largest_acknowledged);
```

Comment:
I don't think that we should be doing handshake drop until we have HANDSHAKE_DONE.  Otherwise we're just going to have to redo this.

Reply:
To be clear, I think that this is likely not correct now, or in future when HANDSHAKE_DONE is added, so I don't want to discard keys here and risk problems.  We can safely keep them indefinitely until we get HANDSHAKE_DONE.

Reply:
ok I will remove dropping of handshake keys.